test/server/cards/03_TWI/events/DisruptiveBurst.spec.ts

```diff
@@ -0,0 +1,45 @@
+describe('Disruptive Burst', function() {
+    integration(function(contextRef) {
+        describe('Disruptive Burst\'s ability', function() {
+            it('should apply -1/-1 to all enemy units for this phase', function () {
```

Comment:
nit: can merge the `describe` into the `it` here since there's just one case and just have a top-level `it`